feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListView.kt

```diff
@@ -25,14 +25,11 @@ import com.iteneum.designsystem.utils.TextUtils.TWO
 fun CommunityListView(
     viewModel: CommunityListViewModel = hiltViewModel()
 ) {
-    LaunchedEffect(true){
+    LaunchedEffect(true) {
```

Comment:
Delete the space

Reply:
I can't delete it cause it's part of the reformat code